.changeset/popular-beds-invent.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-search-backend-module-elasticsearch': patch
+---
+
+Index templates can now be configured through configuration under the `search.elasticsearch.indexTemplates`. In addition, the `ElasticSearchSearchEngine.fromConfig` now also accepts a `LoggerService` as the `logger` option as well as a new `translator` option.
```

Comment:
Hm unfortunate that this wasn't already under a `search.engines` prefix or so. But that can always be added as an alias some time down the line of course

Reply:
Yep, will leave that for someone else 😁